cli.py

```diff
@@ -66,8 +66,20 @@ def cli_entry_point(scrape, resume, verify, resume_verify, compile_active, delet
         return
 
     if scrape:
-        if dm is None:
+        if not dm:
```

Comment:
None does evaluate to False, though. Why do we need to insist it be None rather than set to False (if that was even possible in the CLI)?